.gitignore

```diff
@@ -36,7 +36,7 @@
 
 # VSCode
 .vscode/*
-!.vscode/launch.json
+.vscode/launch.json
```

Comment:
Did you intend to change this? The `!` means `.vscode/launch.json` isn't ignored despite the `.vscode/*` line above. If we should be ignoring the file then the line should be removed altogether as the line above will already catch it.

Reply:
Intended to change to prevent placing my local launch.json under source control, I'll remove the line.